diff --git a/node_modules/chrome-launcher/dist/random-port.js b/node_modules/chrome-launcher/dist/random-port.js
index ef93f4b..08979e7 100644
--- a/node_modules/chrome-launcher/dist/random-port.js
+++ b/node_modules/chrome-launcher/dist/random-port.js
@@ -13,7 +13,9 @@ const http_1 = require("http");
 function getRandomPort() {
     return new Promise((resolve, reject) => {
         const server = http_1.createServer();
-        server.listen(0);
+        const PORT = 9222
+        console.info(`use ${PORT} to remote debug chrome.`)
+        server.listen(PORT);
         server.once('listening', () => {
             const { port } = server.address();
             server.close(() => resolve(port));
