diff --git a/dist/shared/index.js b/dist/shared/index.js
index 62643d269f298e5929441db611caa20a2e90de50..3f5c04bc8b2e04720d285bc560e9e6576aa34a1c 100644
--- a/dist/shared/index.js
+++ b/dist/shared/index.js
@@ -3998,6 +3998,7 @@ constructor(_opts) {
  * @returns {FSWatcher} for chaining
  */
 add(paths_, _origAdd, _internal) {
+  if(typeof paths_ === "string" && paths_.startsWith("/")) return this;
   const {cwd, disableGlobbing} = this.options;
   this.closed = false;
   let paths = unifyPaths(paths_);